docsets/pyspark/docset.json

```diff
@@ -12,6 +12,14 @@
         "GraphX"
     ],
     "specific_versions": [
+        {
+            "author": {
+                "name": "Tomas Kunka",
+                "link": "https://twitter.com/TomasKunka"
+            },
+            "version": "2.4.0",
+            "archive": "versions/2.4.0/pyspark.tgz"
```

Comment:
You need to actually add this archive. Moving the `txt` file doesn't do anything.